build/index.html

```diff
@@ -230,6 +230,15 @@ <h2 class="ons-cookies-banner__title ons-u-mb-xs">Cookies on StaticWebsiteTempla
                         <li class="ons-navigation__item ">
                            <a class="ons-navigation__link" href="/page-3" >Page 3</a>
                         </li>
+                        <li class="ons-navigation__item ">
+                           <a class="ons-navigation__link" href="/mkdocs_page1" >Mkdocs Page 1</a>
+                        </li>
```

Comment:
Could we have the Navigation so that it renders as: External Markdown, Local Markdown, Python Docstrings so it is more obvious what the examples are?

Reply:
done